erpnext/patches/v13_0/set_billed_amount_in_returned_dn.py

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) 2022, Frappe and Contributors
+# License: GNU General Public License v3. See license.txt
+
+import frappe
```

Comment:
This patch will try to update Delivery Notes right from the very start. The books must be frozen for most of the users for the previous fiscal years so not a good idea to change those now and this patch will also take a lot of time for some users. Hence I'll suggest removing this patch and handle on a case to case basis if needed on the site

Also, please add an example with some actual values like quantities and amount so that the fix can be easily tested or replicated